### ISSUE TITLE
Long notes cut off early

### ISSUE THREAD

**Issue by erinic04** [musescore/MuseScore#22691]
### Issue type

General playback bug

### Bug description

When you have a long note (its just more visible for long notes) it will cut off the note early and not play the whole duration of the note. For a 31 measure long note at 120 bpm it cuts off about 7 beats early from what I've seen

### Steps to reproduce

1. Make new score
2. Enter in a long note
3. Observe the note ending early

### Screenshots/Screen recordings

https://github.com/musescore/MuseScore/assets/21023353/8da12519-7881-4444-94cd-2b3d0de86cf8

https://github.com/musescore/MuseScore/assets/21023353/782af74f-187c-4778-8d3a-b8cbcdce6a26

https://github.com/musescore/MuseScore/assets/21023353/aad22f77-d1c9-4ce2-900a-5956b6226cab

### MuseScore Version

OS: Windows 10 Version 2009 or later, Arch.: x86_64, MuseScore version (64-bit): 4.2.1-240230937, revision: github-musescore-musescore-d757433

### Regression

I don't know

### Operating system

Reported on Windows 11

**Comment by DaddyLudwig** [comment]
I think this is by design that the last 5% isn't played cause that helps to separate out notes with short note values and/or fast tempos.

**Comment by erinic04** [comment]
Even if it is by design, it shouldn't be a percentage, rather it should be a fixed note value so you don't run into problems with really long notes.

**Comment by MarcSabatella** [comment]
Indeed.  The 95% was an experiment from long ago, implemented for piano and flute only originally, to simulate the effect of a slur (notes under a slur were not shortened.  Percentage was used because that was the only way to represent that in the old instruments.xml that controlled articulations.

So I assume the current behavior is artifact of that, and definitely should be changed so that notes get full value by default.  A shortening of the last note under a slur is reasonable, but as noted, it shouldn’t be a percentage.

**Comment by zacjansheski** [comment]
This only seems to be an issue with specific MSBasic instruments.

I'm guessing theres more but I believe the organ and synthesizers instruments are the main culprits

**Comment by DmitryArefiev** [comment]
Still occurs after #25187
